### PR TITLE
Update faq.rst

### PR DESCRIPTION
**Pull request by Pri3st**
Added a workaround for WSL 2 consuming high amounts of RAM, leading to reduced performance of the host machine.

### CODE REVIEW

File: source/getting-started/faq.rst [hunk 1]
```diff
@@ -102,4 +102,15 @@ How to retrieve your desktop login details ?
 
 The container's root password can be obtained with ``exegol info <container>`` (i.e. this is needed when using the :doc:`desktop </the-exegol-project/python-wrapper>` feature)
 
-.. TODO: add a note, when the Desktop feature is in prod, that explains the ups and dows of X11 vs. Desktop mode.
\ No newline at end of file
+WSL 2 consumes massive amounts of RAM and CPU power. How can I deal with this issue ?
+============================================
```
Comment (ShutdownRepo) [inline source/getting-started/faq.rst:106]: We need equal number of `=` to len(title)

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]
Thank you, one change and we'll be good to go, thank you @Pri3st !!

**Comment by Pri3st** [comment]
Hey @ShutdownRepo , I will also need to add another workaround to deal with WSL2 getting too much disk space. I have found a solution with the `diskpart` tool which can significantly free up space because WSL2 seems to be increasingly claiming disk space that cannot be controlled through the `.wslconfig` file. In my case, after an Exegol image update, WSL2 claimed an additional 60GB  of disk space (the previous image version was deleted) so I had to troubleshoot a bit to resolve this.

I think it will be a good addition to this PR. I will add the workaround and let you know.

**Comment by Pri3st** [comment]
@ShutdownRepo the FAQ section is now complete, you can review it at your convenience.

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Looks good to me, thank you for your contribution to the Exegol project @Pri3st !!

**Pull request merged**